webhook/webhook.go

```diff
@@ -126,7 +125,6 @@ func New(
 		Options:     *opts,
 		Logger:      logger,
 		synced:      cancel,
-		gracePeriod: network.DefaultDrainTimeout,
```

Comment:
We should set to `opts.GracePeriod` the previous default value `network.DefaultDrainTimeout` when `opts.GracePeriod` isn't set.